build.html

```diff
@@ -6,12 +6,12 @@
       <div class="row">
         {{#if showTotalEntries}}
         <div class="col-xs-6 {{#unless autoRefresh}}col-xs-offset-3{{/unless}}">
-          <div class="chart-total"><i class="fa fa-tachometer"></i> Total<br><span class="total">&mdash;</span></div>
+          <div class="chart-total" data-translate="[html]widgets.chart.line.total"><i class="fa fa-tachometer"></i> Total<br><span class="total">&mdash;</span></div>
```

Comment:
Let's keep the scope to only the "Total" string. The same for the "Last updated" string

```suggestion
          <div class="chart-total"><i class="fa fa-tachometer"></i> <span data-translate="widgets.chart.total">Total</span><br><span class="total">&mdash;</span></div>
```